docs/testbook.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+# Generate the testbook for the functional tests
+#
+# To run this file you will need testbook installed into your DOAJ env
+#
+# source [doaj env]/bin/activate
+# git clone git@github.com:CottageLabs/testbook.git
+# cd testbook
+# pip install -r requirements.txt
+#
+# ~~Testbook:Script->Testbook:Technology~~
+
+BRANCH=$(git branch 2>/dev/null | grep '^*' | colrm 1 2)
+
+TESTBOOK_URL="https://doaj.github.io/doaj-docs/$BRANCH/testbook"
+DOAJ_URL="http://testdoaj.cottagelabs.com/"
+DOAJ_TEST_FILEBASE="https://raw.githubusercontent.com/DOAJ/doaj/$BRANCH/doajtest"
+
+BASE_DIR=$(dirname $(cd $(dirname "${BASH_SOURCE[0]}") && pwd))
+DOAJ_DOCS="$(dirname $BASE_DIR)/doaj-docs"
+
+while getopts 'd:b:t:f:' OPTION; do
+  case "$OPTION" in
+    d)
+      DOAJ_DOCS="$OPTARG"
+      ;;
+    b)
+      TESTBOOK_URL="$OPTARG"
+      ;;
+    t)
+      DOAJ_URL="$OPTARG"
```

Comment:
oh, fine!

Reply:
I also didn't read this far before making my comment above